mmomni/ansible/playbooks/jitsi/videobridge.sip-communicator

```diff
@@ -0,0 +1,11 @@
+org.ice4j.ice.harvest.DISABLE_AWS_HARVESTER=true
```

Comment:
You may want to migrate to the new config: https://github.com/jitsi/docker-jitsi-meet/commit/130eb551a7666a9a143343e4ac7e707e3d6c6c02#diff-7a9436b3d0fae143a1b4d7ccad86a25bd6a50d85a31745a582ba32f466cd7e40

Also, enabling WS for the bridge channel is recommended: https://github.com/jitsi/docker-jitsi-meet/commit/ad5625bb0990ee20f599d0e73eac06df115e2fb4#diff-7a9436b3d0fae143a1b4d7ccad86a25bd6a50d85a31745a582ba32f466cd7e40 (and a couple of follow-up commits)